cases-SEP24/transaction.test.js

```diff
@@ -82,6 +82,12 @@ describe("Transaction", () => {
       jwt: jwt,
     });
     await checkTransactionResponse({ json: json, isDeposit: true });
+    const notExpectedJson = json;
```

Comment:
I think we can remove this file's changes right? The first `checkTransactionResponse()` call in each test will raise an error if the conditions tested below would fail.

Reply:
Ah I see your point, yeah I'll revert `cases-SEP24/transaction.test.js`'s changes.